### DISCUSSION TITLE
gum has the right version on windows (scoop) ?

### DISCUSSION THREAD

**Discussion by khengyun** [charmbracelet/gum#27] [Ideas]
it would be great if it combined with oh-my-posh

**Comment by BlueFalconHD** [comment]
What do you mean?

**Comment by caarlos0** [comment]
gum is not in scoop yet.

the `gum` available there is a maven wrapper: https://scoop.sh/#/apps?q=gum&s=0&d=1&o=true

**Reply by BlueFalconHD** [reply]
Maybe compile from source, or use the gum releases for windows.
![image](https://user-images.githubusercontent.com/72631767/182426199-339a2bba-ed5f-4d57-8af6-391fa7530e67.png)

**Comment by beltagyy** [comment]
For Windows users, Scoop typically tracks the latest stable release. You can verify the installed version with gum --version and compare it to the latest release on the GitHub Releases page. If Scoop is behind, running scoop update gum usually brings it up to date.
